solidity/dashboard/src/pages/coverage-pools/CoveragePoolPage.jsx

```diff
@@ -71,6 +72,43 @@ const CoveragePoolPage = ({ title, withNewLabel }) => {
     <PageWrapper title={title} newPage={withNewLabel}>
       <CheckListBanner />
 
+      <section className="tile">
+        <Timeline>
+          <Timeline.Element>
+            <Timeline.ElementBreakpoint active>1</Timeline.ElementBreakpoint>
+            <h4 className="text-violet-80">Deposit your tokens</h4>
+            <span className="text-grey-60">
+              There is no minimum KEEP amount for your deposit and no minimum
+              time lock.
+            </span>
+          </Timeline.Element>
+
+          <Timeline.Element>
+            <Timeline.ElementBreakpoint active>2</Timeline.ElementBreakpoint>
+            <h4 className="text-violet-80">Withdraw deposit </h4>
+            <span className="text-grey-60">
+              <strong>Withdrawing is a two step action.</strong>&nbsp;From the
+              initiation of your withdrawal to claiming your tokens there is a
+              21 day cooldown period in which your tokens are still accumulating
+              rewards but are also subject to risk to cover for a hit.
+            </span>
+          </Timeline.Element>
+
+          <Timeline.Element>
+            <Timeline.ElementBreakpoint active>3</Timeline.ElementBreakpoint>
+            <h4 className="text-violet-80">Claim tokens</h4>
+            <span className="text-grey-60">
+              <strong>
+                You have a 2 day grace period to claim your tokens and rewards.
+              </strong>
```

Comment:
I think we can extract the `<h4>` element to a separate component like `<Timeline.ElementTitle>`.

I was thinking that we could also to the same with the `<span>` and extract it to something like ` <Timeline.ElementDescription>` since both Timelines (the one on `How it works` subpage and the one in modal window) are pretty similar so I don't think we need to put it all as `children` element for now (but we can also keep the possibility to specify `children` here and keep `title` and `description` optional as it might be more reusable in the future.

Reply:
TBH I think it's unnecessary. Passing children give us the most flexibility. Based on few other libs they define only a kind of `Content` component where we can pass children. In our cases, the timelines on the `How it works page` and in `modal` are slightly different so I think it's hard to define common styles. Probably on another page, the timeline would be different.

Also, I would consider getting rid of common styles(like violet borders) for `Timeline.Element` component- I think it should be defined by parent component inside `Timeline.Element` component.

Reply:
I see your point and I think you are right. Let's keep it that way then.

And i also agree with the seconds paragraph too.